deploy/helm/grafana-operator/values.yaml

```diff
@@ -6,18 +6,27 @@ namespaceScope: false
 leaderElect: false
 
 # -- Sets the WATCH_NAMESPACE environment variable,
-# it defines which namespaces the operator should be listening for.
-# By default it's all namespaces, if you only want to listen for the same namespace as the operator is deployed to look at namespaceScope.
+# Defines namespaces the operator should be listening for.
+# By default it's all namespaces, if you want to limit the Operator to its own namespace, instead set namespaceScope: true.
 watchNamespaces: ""
 
 # -- Sets the WATCH_NAMESPACE_SELECTOR environment variable,
-# it defines which namespaces the operator should be listening for based on label and key value pair added on namespace kind.
+# Defines namespaces the operator should be listening for based on label and key value pair added on namespace kind.
```

Comment:
If you take a look at generated docs, you'll see that two lines get merged, that's why the previous wording used lower-case started from it. I think you should revert this change.